src/python/bin/aqueduct

```diff
@@ -43,6 +43,9 @@ welcome_message = """
 Your API Key: %s
 
 The Web UI and the backend server (v%s) are accessible at: http://%s:%d/login?apiKey=%s
+
+Aqueduct collects usage data to improve its services.
```

Comment:
I personally feel a bit more clarification might help having user more willing to use this feature. Telling user what's been collected and what they are used for is in fact enforced by regulation for larger apps:
"Aqueduct collects usage data to improve our service performance and scalability. We only collect the frequency of each REST endpoint and their performance stats. We do NOT collect any personal information or anything related to data processed in your workflows.
To disable data collection, run aqueduct service by using `...`"
Alternatively, the long sentence can go to a documentation and we put a link here (maybe as a follow up)

Reply:
Yes, I'm going to make a GitBook page that describes in more detail which stats we are collecting, and add a link to here. Will ship them together.